main.js

```diff
@@ -133,6 +150,19 @@ define(function (require, exports, module) {
         };
     }
     
+    /**
+     * Apply the whitespace colors.
+     * This does not overwrite styles already defined by a theme.
+     */
+    function _applyColors() {
+        // Compile the CSS from the template
+        var template = _.template(stylesTemplate),
```

Comment:
You can create this `_.template` when the extension loads so that the template does get parsed each time.  This routine will basically become:

``` javascript
 // Update the styling with the changes
_styleInline.text(template(_preferences.get("colors")));
```